doc/classes/RichTextLabel.xml

```diff
@@ -329,9 +329,6 @@
 			The raw text of the label.
 			When set, clears the tag stack and adds a raw text tag to the top of it. Does not parse BBCodes. Does not modify [member bbcode_text].
 		</member>
-		<member name="text_direction" type="int" setter="set_text_direction" getter="get_text_direction" enum="Control.TextDirection" default="0">
-			Base text writing direction.
-		</member>
```

Comment:
I guess this was a rebase mistake and is actually added by #42595 @bruvzg?

Reply:
Yes, it should be in #42595.